data/Building/reply.js

```diff
@@ -95,4 +100,60 @@ async function reply(buildingId, parentCommentId, posterId, message) {
   ).exec();
 }
 
-module.exports = reply;
+/**
+ * Modify a reply to a comment
+ * @param {String} buildingId
+ * @param {String} parentCommentId
+ * @param {String} posterId
+ * @param {String} newMessage
+ * @param {String} oldMessage
+ * @return {Object}
+ */
+async function modifyReply(buildingId, parentCommentId, posterId, newMessage, oldMessage) {
```

Comment:
you use the parentCommentId here, but you should also use the replyId of the reply you're modifying